demos/shared-ros/Makefile

```diff
@@ -0,0 +1,16 @@
+all: ros-app/*.snap
```

Comment:
this rogued itself in, right?

Reply:
Actually no, I got tired of the manual steps to build the consumer, so decided to wrap it in a Makefile. I doubt this is the last time it'll be useful.